content/300-guides/300-upgrade-guides/950-upgrading-to-referential-actions.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Upgrading to referential actions (2.26.0)'
+metaTitle: 'Upgrading to referential actions (2.26.0) (Preview guide)'
+metaDescription: 'Upgrading your project to use referential actions.'
+preview: true
+---
+
+<TopBlock>
+
+These guides explain how to adopt the 2.26.0 preview feature, referential actions.
+
+<Admonition type="warning">
+
+**If you have a project that is running version 2.25.0 and you then upgrade to 2.26.0 and enable the preview feature, using the `delete()` or `deleteAll()` methods may results in deletion of data where once it would not have.**
+
+</Admonition>
+
+The feature can be accessed by:
+
+1. [Upgrading to version v2.26.0 of Prisma](../../../../guides/upgrade-guides/upgrading-to-latest).
+2. Adding `referentialActions` to the `previewFeatures` field of the Prisma Client [`generator`](../../../../reference/api-reference/prisma-schema-reference#generator) block in the `prisma.schema` file.
+
+```prisma file=schema.prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["referentialActions"]
+}
+```
```

Comment:
```suggestion
  ```prisma file=schema.prisma
  generator client {
    provider        = "prisma-client-js"
    previewFeatures = ["referentialActions"]
  }
  ```
```
Indentation so this is part of the list which is belongs to.